news/_posts/releases/2020-04-20-release-note-1765.md

```diff
@@ -0,0 +1,102 @@
+---
+layout: post
+title:  "Release Of PrestaShop 1.7.6.5"
+subtitle: "Maintenance version of the 1.7.6.X branch"
+date:   2020-04-20 16:30:00
+authors: [ PrestaShop ]
+image: /assets/images/2020/05/1765meta.jpg
+icon: icon-checked
+tags:
+ - 1.7
+ - 1.7.6.x
+ - version
+ - patch
+ - releases
+---
+
+
+As announced a few months ago, the PrestaShop core team has decided to make a forthright patch release process and deliver patch versions on a more regular basis, every time it’s needed. So here we are, 6 weeks after the release of 1.7.6.4, PrestaShop 1.7.6.5 is now available! 
+
+![1.7.6.5 is available!](/assets/images/2020/04/1765banner.jpg)
+
+This maintenance release is a bit special as it does not only fix regressions found on version 1.7.6.0 to 1.7.6.4, but it also has put a focus on fixing many security issues, from 1.5, 1.6 and 1.7 versions. This is a result of a huge work on security which has been started a few weeks ago to ensure more security on the PrestaShop software. In the near future, PrestaShop will focus more and more on security to ensure that no security breaches, even minor ones such as permission issues, are left out in the core. 
+
+As this patch fixes several security issues, we highly recommend to upgrade your shop as soon as possible. Of course, as always, don’t forget to backup before.
+
+Reminder:  the 1-Click Upgrade module’s latest version is [v4.10.1](https://github.com/PrestaShop/autoupgrade/releases/tag/v4.10.1), don’t forget to upgrade it.
+
+
+## Main fixes
+
+Below are listed the 7 regressions that were found and fixed in this version, impacting both front-office and back-office.
+
+Front-office regressions:
+
+- When editing an address both in the customer account and checkout, a new address was created instead of replacing it - [#18100](https://github.com/PrestaShop/PrestaShop/issues/18100) and [#18072](https://github.com/PrestaShop/PrestaShop/issues/18072)
+- Canonical redirects for products with combinations no longer worked, which could cause duplicate content [#18279](https://github.com/PrestaShop/PrestaShop/issues/18299)
```

Comment:
#18299

Reply:
nice catch I did a mistake while pasting the links, I guess.
we should write them in markdown in the google doc too, this is very tricky with a release note like with a looot of links. I will manage that.

Reply:
done